src/objects/zcl_abapgit_oo_class_new.clas.abap

```diff
@@ -303,18 +303,21 @@ CLASS ZCL_ABAPGIT_OO_CLASS_NEW IMPLEMENTATION.
 
   METHOD update_source_index.
 
-    DATA li_index_helper TYPE REF TO if_oo_source_pos_index_helper.
+    "    dynamic invocation IF_OO_SOURCE_POS_INDEX_HELPER doesn't exist in 702.
```

Comment:
if it does not exist, this code will cause a dump? Suggest instead catching the error and returning from method, it might be okay not to update the index

Reply:
You're right. I'll fix.